docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/ckeditor/Ckeditor.scss

```diff
@@ -98,6 +98,7 @@ CSS modules.
 @import "~Core/styles/checkbox";
 @import "~Core/styles/popups";
 @import '~Core/styles/tableSort';
+@import '~Core/styles/sbir';
```

Comment:
Can you remove the reference to SBIR in commit 1f927bdd27c8de99f539c9f523bb4670859c77d1. This is not part of ticket 3311.